### PR TITLE
chef 12 windows 2012, rubocop

### PR DESCRIPTION
**Pull request by heathsnow**
@sweitzel74 please review

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -7,14 +7,12 @@ provisioner:
   name: chef_zero
 
 platforms:
-  - name: ubuntu-14.04
+  - name: ubuntu-14.04-virtualbox
     driver:
-      box: virtualbox_ubuntu-14.04_chef-12.4.1_1.0.7
-      box_url: http://vagrantboxes.hq.daptiv.com/vagrant/boxes/virtualbox_ubuntu-14.04_chef-12.4.1_1.0.7.box
-  - name: windows2012r2
+      box_url: 'http://artrepo.daptiv.com:8081/artifactory/vagrant-local/virtualbox_ubuntu-14.04_chef-12.4.1_1.0.13.box'
```
Comment (sweitzel74) [inline .kitchen.yml]: Parameterize in here

File: .kitchen.yml [hunk 2]
```diff
@@ -7,14 +7,12 @@ provisioner:
...
+  - name: windows2012r2_chef12
     driver:
-      box: virtualbox_windows-2012r2_chef-12.5.1_1.0.23
-      box_url: http://artrepo.daptiv.com:8081/artifactory/vagrant-local/virtualbox_windows-2012r2_chef-12.5.1_1.0.23.box
+      box_url: 'http://artrepo.daptiv.com:8081/artifactory/vagrant-local/virtualbox_windows-2012r2_chef-12.5.1_1.0.23.box'
```
Comment (sweitzel74) [inline .kitchen.yml]: Parameterize in here

File: metadata.rb [hunk 3]
```diff
@@ -1,14 +1,14 @@
-name             'google-chrome'
-maintainer       'Daptiv Solutions, LLC'
+name 'google-chrome'
+maintainer 'Daptiv Solutions, LLC'
 maintainer_email 'sneal@daptiv.com'
```
Comment (sweitzel74) [inline metadata.rb]: No more sneal

File: metadata.rb [hunk 4]
```diff
@@ -1,14 +1,14 @@
-name             'google-chrome'
-maintainer       'Daptiv Solutions, LLC'
+name 'google-chrome'
+maintainer 'Daptiv Solutions, LLC'
```
Comment (sweitzel74) [inline metadata.rb]: Changepoint

**Comment by heathsnow** [comment]
@sweitzel74 updated, and travis file/badge added.

**Comment by sweitzel74** [comment]
:shipit:

**Pull request merged**